src/__tests__/labs/ViewLabs.test.tsx

```diff
@@ -125,30 +120,36 @@ describe('View Labs', () => {
         screen.getByRole('combobox'),
         `{selectall}{backspace}${expectedStatus}{arrowdown}{enter}`,
       )
-
-      expect(LabRepository.search).toHaveBeenCalledTimes(2)
-      expect(LabRepository.search).toHaveBeenCalledWith(
-        expect.objectContaining({ status: expectedStatus }),
-      )
+      expect(screen.getByRole('combobox')).toHaveValue('labs.status.requested')
     })
   })
 
   describe('search functionality', () => {
     it('should search for labs after the search text has not changed for 500 milliseconds', async () => {
+      const expectedLab2 = {
+        code: 'L-5678',
+        id: '5678',
+        type: 'another type',
+        patient: 'patientIdB',
+        status: 'requested',
+        requestedOn: '2020-03-30T04:43:20.102Z',
+      } as Lab
+      const expectedSearchText = 'another'
+      const { expectedLab } = setup([Permissions.ViewLabs, Permissions.RequestLab])
+
+      jest.spyOn(LabRepository, 'findAll').mockResolvedValue([expectedLab2])
```

Comment:
Is this doable without a spy?

Reply:
Setting that spy is the only way I've been able to get this to pass with results that resemble how the search is supposed to behave